tracker/view/add.py

```diff
@@ -17,13 +18,15 @@
 from tracker.model.enum import affected_to_status
 from tracker.model.enum import highest_severity
 from tracker.user import reporter_required
+from tracker.user import user_can_edit_issue
 from tracker.util import multiline_to_list
 
 ERROR_GROUP_WITH_ISSUE_EXISTS = 'The group AVG-{} already contains {} for the package {}'
 ERROR_OPEN_GROUP_EXISTS = 'The group AVG-{} already has open issues for the package {}'
 CVE_MERGED = 'Merged existing {} with the provided data'
 CVE_MERGED_PARTIALLY = 'Failed to fully merge {}, check the following fields: {}'
 ERROR_UNMERGEABLE = 'Unmergeable field, old value shown'
+ERROR_ISSUE_EXISTS = 'Insufficient permissions to edit existing {}'
```

Comment:
nice catch, however i think the variable name and the actual message doesn't really cover what its used for.
we should mention that its about Insufficient permissions to edit a CVE that is referenced by an already published advisory!
maybe reflect in `ERROR_ISSUE_REFERENCED_BY_ADVISORY` or something similar

Reply:
Maybe a second commit could make use of this error flash in the other place as well?

Reply:
> maybe reflect in `ERROR_ISSUE_REFERENCED_BY_ADVISORY` or something similar

Sure, I renamed it according to your suggestion.

> Maybe a second commit could make use of this error flash in the other place as well?

Sounds good to me, added.

Reply:
nah, i also meant the text :cat: I think it doesn't quite explain _why_ it can't be edited.
like
```
Insufficient permissions to edit {} that is referenced by an already published advisory!
```
What do you think?

Reply:
The reason for the somewhat vague error message is that I didn't necessarily want to describe the internal logic of `user_can_edit_issue` somewhere else in case it changes. Given that it is probably unlikely to change in the foreseeable future though, I am happy to expand the wording like you suggested.

Reply:
yeah, totally see your point here. if we ever touch that we may want to change bool return to an enum or similar.